### PR TITLE
Add GLBC ingress resource example

### PR DESCRIPTION
**Pull request by kyessenov**

### CODE REVIEW

File: k8s/esp_echo_gke_ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-echo
+    spec:
+      containers:
+      - name: esp
+        image: b.gcr.io/endpoints/endpoints-runtime:1.0
```
Comment (wlu2016) [inline k8s/esp_echo_gke_ingress.yaml]: I assume we should wait until we have the GA version released before  I merge this PR?

we should have a stable tag as discussed. Otherwise, we would need to keep updating this file whenever we cut a new major.minor

File: k8s/esp_echo_gke_ingress.yaml [hunk 2]
```diff
@@ -40,7 +40,7 @@ spec:
     spec:
       containers:
       - name: esp
-        image: b.gcr.io/endpoints/endpoints-runtime:1.0
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
```
Comment (wlu2016) [inline k8s/esp_echo_gke_ingress.yaml]: do we have "latest" tag now?
and please also document this is just for samples, which versions should be used for production. We probably should have a page about released versions for production use

File: k8s/README.md [hunk 3]
```diff
@@ -202,6 +202,36 @@ To send a request to the API
...
+You can check the status of your ingress using the following command:
+
+  ```
+  kubectl describe ingress
+  ```
+
+Once you obtain an external address, we can send a request to the echo backend:
+
+  ```
+  curl -d '{"message":"hello world"}' -H "content-type:application/json" http://${INGRESS_IP_ADDRESS}/echo?key=${ENDPOINTS_KEY}
```
Comment (lookuptable) [inline k8s/README.md]: "ENDPOINTS_KEY" should be "API_KEY"?

Comment (kyessenov) [inline k8s/README.md]: ENDPOINTS_KEY is the API key here (see above), isn't it?

File: k8s/esp_echo_gke_ingress.yaml (hunk 2 above)
Comment (kyessenov) [inline k8s/esp_echo_gke_ingress.yaml]: We will need to go over and change all images in the samples. Probably do it in another commit?

File: k8s/README.md (hunk 3 above)
Comment (lookuptable) [inline k8s/README.md]: I see. Didn't notice it's mentioned above.

It's okay to use ENDPOINTS_KEY in your PR since it's not newly introduced. But we probably should change it to API_KEY. I never see the usage of ENDPOINTS_KEY anywhere else. I'll do a PR after yours if you agree.

File: k8s/esp_echo_gke_ingress.yaml (hunk 2 above)
Comment (wlu2016) [inline k8s/esp_echo_gke_ingress.yaml]: ping me once we figured out the release page to refer here. We should at least document :latest is for sample/test/dev only, not for production use

File: k8s/esp_echo_gke_ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-echo
+    spec:
+      containers:
+      - name: esp
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
```
Comment (qiwzhang) [inline k8s/esp_echo_gke_ingress.yaml]: tag should be 1 to use the latest version 1

File: k8s/esp_echo_gke_ingress.yaml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        args: [
+          "-p", "8080",
+          "-a", "127.0.0.1:8081",
+          "-s", "SERVICE_NAME",
+          "-v", "SERVICE_VERSION",
```
Comment (qiwzhang) [inline k8s/esp_echo_gke_ingress.yaml]: SERVICE_CONFIG_ID

File: k8s/esp_echo_gke_ingress.yaml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-echo
+    spec:
+      containers:
+      - name: esp
+        image: b.gcr.io/endpoints/endpoints-runtime:1
```
Comment (wlu2016) [inline k8s/esp_echo_gke_ingress.yaml]: fix b.gcr.io to gcr.io/endpoints-release

Let me know if this pr ready for merge after this change

Comment (kyessenov) [inline k8s/esp_echo_gke_ingress.yaml]: PTAL
